angularjs-portal-home/src/main/webapp/my-app/layout/services.js

```diff
@@ -96,6 +99,27 @@ define(['angular', 'jquery'], function(angular, $) {
             if (folders && 0 < folders.length) {
               result.layout = folders[0].portlets;
             }
+          } else if ($.isArray(data.layout.navigation.tabs)) { // v4-3
```

Comment:
Won't this throw an error if we're using layoutDoc, since data.layout.navigation is undefined?